acceptance/steps/manifest.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2024 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package steps
+
+import (
+	"context"
+	"os"
+
+	"github.com/cucumber/godog"
+	framework "github.com/redpanda-data/redpanda-operator/harpoon"
+	"github.com/stretchr/testify/require"
+)
+
+func iApplyKubernetesManifest(ctx context.Context, t framework.TestingT, manifest *godog.DocString) {
+	file, err := os.CreateTemp("", "manifest-*.yaml")
```

Comment:
Because we don't have access to a real `testing.T` because of... `godog` and them not exposing `testing.T` instances apart from the logging/Fail/Skip methods -- we'd need to do an upstream PR for that.